pkg/cli/admin/release/new_test.go

```diff
@@ -0,0 +1,97 @@
+package release
+
+import (
+	"strings"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	imageapi "github.com/openshift/api/image/v1"
```

Comment:
Nit: `imagev1` is the correct alias for these APIs

Reply:
And also you're mixing openshift and k8s ones, I'd keep each in a separate group :wink: